ProcessLib/LiquidFlow/LiquidFlowLocalAssembler-impl.h

```diff
@@ -121,7 +121,11 @@ void LiquidFlowLocalAssembler<ShapeFunction, IntegrationMethod, GlobalDim>::
 
     if (gravitational_axis_id >= 0)
     {
-        local_b.noalias() -=
+        // Note: Since perm, K, is a scalar number in this function,
+        // (gradN)*K*V becomes K*(grad N)*V. With the gravity vector of V,
+        // the simplification of (grad N)*V is the gravitational_axis_id th
+        // column of the transpose of (grad N) multiplied with rho_g.
+        local_b.noalias() +=
             fac * sm.dNdx.transpose().col(gravitational_axis_id) * rho_g;
```

Comment:
It is for (grad(N))^T * V. If use sm.dNdx.row(), then (grad(N))^T * V should be written as
```
sm.dNdx.row(gravitational_axis_id).transpose() * rho_g
```
and it is less understandable.

PS: There is a ctest GravityDriven for this implementation.